assets/scripts/embedding_tests_dev_0.0.1.min.html

```diff
@@ -8,21 +8,27 @@ <h2>v0.0.1 (minified)</h2>
 
 <script src="/assets/scripts/oppia-player-0.0.1.js"></script>
 <script src="/assets/scripts/embedding_tests.js"></script>
+<script src="../../third_party/static/jquery-3.2.1/jquery.min.js"></script>
+
+<form action="#" onsubmit="update()">
```

Comment:
Call this something like onChangeExpId() instead of update()

Reply:
done.